code/modules/nifsoft/software/13_soulcatcher.dm

```diff
@@ -461,6 +462,8 @@
 		if(istype(HP) && HP.nif && HP.nif.flag_check(NIF_O_SCOTHERS,NIF_FLAGS_OTHER))
 			var/datum/nifsoft/soulcatcher/SC = HP.nif.imp_check(NIF_SOULCATCHER)
 			SC.catch_mob(H)
+		else if(H.nif && H.nif.flag_check(NIF_SC_BACKUPS))
+			SStranscore.m_backup(H,0) //If dying inside a pred who lacks SC, let ours do a backup if enabled before it's gone for good.
```

Comment:
While this is obvious and logical for a thing to exist and I'm totally down for it, I sort of wonder if people are doing to use this as a thing to point to as to why they have no CMD. Probably warrants admin discussion.